power app sample/readme.md

```diff
@@ -0,0 +1,76 @@
+![banner](https://i.ibb.co/QQCVq4m/banner.png)
+
+## Overview
+
+This is a fully packaged solution for the **D365 Fraud Protection Portal Admin App** that serves multiple purposes:
+
+- Demonstrate the extensibility capabilities of Fraud Protection Event Tracing and provide an open-source reference for those creating their own custom Power App.
+- Provide IT Engineers and Fraud Managers with a toolset for actively monitoring latency measurements for their Fraud Protection instance as well as maintaining an interactive audit log of Fraud Protection portal related events.
+
+The app will work directly with Fraud Protection **Audit Events** and **Latency Events** data.
+
+## Prerequisites
+
+- Azure subscription
+- Event Hub Namespace
+- Two Event Hubs
+- Dynamics 365 Fraud Protection event tracing subscriptions connected to your two Event Hubs (one for **Audit Events**, one for **Latency Events**)
+- Power Apps subscription
+
+For help setting up the above, check the documentation in **Reference Links** in the following section.
+
+## Reference Links
+
+[Dynamics 365 Fraud Protection Event Tracing Documentation](https://docs.microsoft.com/en-us/dynamics365/fraud-protection/event-tracing)
+
+## Import Solution
+
+1. Download and save the `D365FraudProtectionPortalAdminPowerAppSolution.zip` file in this repository to your computer.
```

Comment:
I went to Power Apps and tried importing the zip.  It worked, but I was wondering if you could change/remove some of the publisher details?  e.g. you probably don't want your phone number and gmail account in there.  Also, not sure if you can make it more "official" by listing Microsoft as the publisher or something?  No worries either way on that, but at least remove your personal info sans name.  Also, is it possible to spell our "Dynamics 365" instead of "D365"?
![image](https://user-images.githubusercontent.com/165767/91142662-fed9e680-e665-11ea-823f-19b3a39e95bc.png)

Reply:
Okay I've added `Microsoft` as a publisher without my personal information. Changed to Dynamics 365 as well.

Reply:
Thanks!